metrics_window: 600
topics_in_order:
  - /benchmarking_suite/timestamps/mqtt_interface_vehicle/packets/out
  - /benchmarking_suite/timestamps/mqtt_interface_vehicle/packets_return/in # 0
export_timestamps: false
export_file: timestamps.csv